packages/tempus/src/Tempus_StepperBDF2_impl.hpp

```diff
@@ -225,7 +222,7 @@ void StepperBDF2<Scalar>::takeStep(
 
     workingState->setSolutionStatus(sStatus);  // Converged --> pass.
     workingState->setOrder(getOrder());
-    this->stepperObserver_->observeEndTakeStep(solutionHistory, *this);
+    //this->stepperObserver_->observeEndTakeStep(solutionHistory, *this);
```

Comment:
Not sure why you needed to comment this out.

Reply:
I'm not exactly sure how a composite observer for BDF methods would work

Reply:
I think it would look like the ExplicitRKObserverComposite, but be specialized for BDF2, e.g., BDF2ObserverComposite.  Probably do not want to tackle that now, but can we leave the previous implementation in place until we get to it?

Reply:
We can tackle this in another PR since it's not critical for any application at the moment

Reply:
Ok, sounds good.  I will approve the review, and set the retest.